README.md

```diff
@@ -109,85 +97,99 @@ The returned value of **every** API call is formed as follows:
-  name: string;
+const { Client } = require('@elastic/elasticsearch')
+const client = new Client({ node: 'http://localhost:9200' })
+
+async function run () {
+  // Let's start by indexing some data
+  await client.index({
+    index: 'game-of-thrones',
+    // type: '_doc', // uncomment this line if you are using Elasticsearch ≤ 6
+    body: {
+      character: 'Ned Stark',
+      quote: 'Winter is coming.'
+    }
+  })
+
+  await client.index({
+    index: 'game-of-thrones',
+    // type: '_doc', // uncomment this line if you are using Elasticsearch ≤ 6
+    body: {
+      character: 'Daenerys Targaryen',
+      quote: 'I am the blood of the dragon.'
+    }
+  })
+
+  await client.index({
+    index: 'game-of-thrones',
+    // type: '_doc', // uncomment this line if you are using Elasticsearch ≤ 6
+    body: {
+      character: 'Tyrion Lannister',
+      quote: 'A mind needs books like a sword needs a whetstone.'
+    }
+  })
+
+  // here we are forcing an index refresh, otherwise we will not
+  // get any result in the consequent search
+  await client.indices.refresh({ index: 'game-of-thrones' })
+
+  // Let's search!
+  const { body } = await client.search({
+    index: 'game-of-thrones',
+    // type: '_doc', // uncomment this line if you are using Elasticsearch ≤ 6
+    body: {
+      query: {
+        match: { quote: 'winter' }
+      }
+    }
+  })
+
+  console.log(body.hits.hits)
 }
+
+run().catch(console.log)
 ```
 
-### Request specific options
-If needed you can pass request specific options in a second object:
-```js
-// promise API
-const result = await client.search({
-  index: 'my-index',
-  body: { foo: 'bar' }
-}, {
-  ignore: [404],
-  maxRetries: 3
-})
+## Install multiple versions
+If you are using multiple versions of Elasticsearch, you need to use multiple versions of the client. In the past, install multiple versions of the same package was not possible, but with `npm v6.9`, you can do that via aliasing.
+
+The command you must run to install different version of the client is:
+```sh
+npm install <alias>@npm:@elastic/elasticsearch@<version>
 ```
-The supported *request specific options* are:
-```ts
-{
-  ignore: [number], // default `null`
-  requestTimeout: number, // client default
-  maxRetries: number, // default `5`
-  asStream: boolean, // default `false`
-  compression: string, // default `false`
-  headers: object, // default `null`
-  querystring: object // default `null`,
-  context: object // default `null`,
-  id: any // default incr. integer
+So for example if you need to install `7.x` and `6.x`, you will run
+```sh
+npm install es6@npm:@elastic/elasticsearch@6
+npm install es7@npm:@elastic/elasticsearch@7
+```
+And your `package.json` will look like the following:
+```json
+"dependencies": {
+  "es6": "npm:@elastic/elasticsearch@^6.7.0",
+  "es7": "npm:@elastic/elasticsearch@^7.0.0"
 }
 ```
+And finally, you will require the packages from your code by using the alias you have defined.
```

Comment:
Omitting "finally" since the next paragraph uses it too and there can only be one final step
```suggestion
You will require the packages from your code by using the alias you have defined.
```